pom.xml

```diff
@@ -43,6 +43,9 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
+        <!-- bump version when performing releases -->
+        <geomesa.release.version>1.1.0-rc.6</geomesa.release.version>
```

Comment:
@elahrvivaz It would be manual. @jnh5y That would be the current project's version, e.g. 1.1.0-rc.7-SNAPSHOT.